src/datasources/transaction-api/transaction-api.manager.ts

```diff
@@ -25,13 +24,9 @@ export class TransactionApiManager implements ITransactionApiManager {
   ) {}
 
   async getTransactionApi(chainId: string): Promise<TransactionApi> {
```

Comment:
I took the opportunity to remove some logging here, I think let me know if you think it was valuable enough or if you want to tackle this in a separate PR 🙂

Reply:
I think it's ok to remove them. This was mostly done in the initial stages of the project and right now I think we have a setup to understand the flow and what might have gone wrong if something happens!